.github/workflows/ci.yml

```diff
@@ -67,7 +67,7 @@ jobs:
 
     - name: Install Raytracing Dependencies
       if: matrix.enable_raytracing == 'true'
-      uses: ./source/.github/actions/ospray-sb-install-dep
+      uses: ./source/.github/actions/ospray-superbuild-install-dep
```

Comment:
Looks like the file is still named ospray-sb-install-dep

Reply:
it is not, it si been renamed

Reply:
Indeed. The diff on mobile doesn't show the file name change